pkgs/development/python-modules/ciscoconfparse/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, passlib
+, dnspython
+, loguru
+, toml
+, ipaddr
+, poetry
+, poetry-core
+, black
+, Fabric
+, pytest
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "ciscoconfparse";
+  version = "1.6.21";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pdplm40vlvx0pc4yxzpvg6drlklz27awj0r0izn7nzrwcj3nbgx";
+  };
+
+  propagatedBuildInputs = [
+    passlib
+    dnspython
+    loguru
+    toml
+    ipaddr
+    poetry
+    poetry-core
+    black
+    Fabric
+    pytest
+    sphinx
+  ];
+
+  doCheck = false;
```

Comment:
Could you add a comment as to why this was done? Also, you should use [`pythonImportsCheck`](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck) when unit tests are disabled, although it can't hurt when they're enabled either.